tests/utils/verification/volumeproperties.go

```diff
@@ -96,6 +96,12 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// CheckVolumeAccessibility returns true if the given volume is accessible
+// from the specified VM; false otherwise.
+func CheckVolumeAccessibility(hostName string, volumeName string) bool {
+	return GetVolumeStatusHost(volumeName, hostName) != ""
```

Comment:
No, this method does not check volume attach/detach status. It only checks whether the volume is available (or visible/accessible) from the given VM.